src/validation.cpp

```diff
@@ -5639,7 +5639,7 @@ util::Result<void> ChainstateManager::ActivateSnapshot(
         bool in_memory)
```

Comment:
`AssumeutxoData::height` is `int`, IMO would be better for them to be the same type, so should  we consider changing it also to `uint32_t`?  this will avoid the issue https://github.com/bitcoin/bitcoin/pull/30516#issuecomment-2273842309 during comparison   or having to explicitly cast one.

Reply:
We would probably have to cast somewhere else then because `nHeight` is `int`.